templates/Pages/home.php

```diff
@@ -36,6 +36,14 @@
                 $error .= '<br />' . $attributes['message'];
             }
         }
+        if ($name === 'debug_kit') {
+            $error = 'Try adding your current <b>top level domain</b> to the
+                <a href="https://book.cakephp.org/debugkit/4/en/index.html#configuration" target="_blank">DebugKit.safeTld</a>
+            config and reload.';
+            if (!in_array('sqlite', PDO::getAvailableDrivers())) {
```

Comment:
```suggestion
            if (!in_array('sqlite', \PDO::getAvailableDrivers())) {
```

There's no `use` statement for the class.

Reply:
Since we are not inside a namespace it should fallback to the global namespace but for the sake of being as specific as possible lets do it.